public/snippet-script.js

```diff
@@ -54,6 +54,7 @@ const currency = popFromURLParameters('currency') || 'EUR';
 const consent = popFromURLParameters('consent') !== 'false';
 const documentDirection = popFromURLParameters('doc-dir') || 'ltr';
 const store = popFromURLParameters('store') || undefined;
+const query = popFromURLParameters('query'); // prevent the query from be included as extra param
```

Comment:
We are not using the query anywhere, so I would just pop the parameter without assigning it to a constant.

```suggestion
popFromURLParameters('query'); // prevent the query from be included as extra param
```